tests/Tests/ORM/Query/NativeQueryTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Query;
+
+use DateTime;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Deprecations\PHPUnit\VerifyDeprecations;
+use Doctrine\ORM\Query\ResultSetMapping;
+use Doctrine\ORM\UnitOfWork;
+use Doctrine\Tests\Mocks\EntityManagerMock;
+use Doctrine\Tests\OrmTestCase;
+
+class NativeQueryTest extends OrmTestCase
+{
+    use VerifyDeprecations;
```

Comment:
Why?

Reply:
@derrabus no reason I think. Removed.

Reply:
I created this file by copying another one, and I forgot to remove that trait